src/librustdoc/html/static/js/search.js

```diff
@@ -4919,11 +4919,12 @@ ${item.displayPath}<span class="${type}">${name}</span>\
         });
     } else if (query.error === null) {
         const dlroChannel = `https://doc.rust-lang.org/${getVar("channel")}`;
+        const docsrsURL = `https://docs.rs/releases/search?query=${encodeURIComponent(query.userQuery)}`;
+        const duckduckgoURL = `https://duckduckgo.com/?q=${encodeURIComponent("rust " + query.userQuery)}`;
```

Comment:
```suggestion
        const duckduckgoURL = `https://duckduckgo.com/?q=${encodeURIComponent("rust lang " + query.userQuery)}`;
```

While we're here we should probably disambiguate so people don't get results about the game (this happens to me sometimes)